### PR TITLE
[MRG] maxfilter ctc channelname bug

### PR DESCRIPTION
**Pull request by SherazKhan**

### CODE REVIEW

**Comment by Eric89GXL** [comment]
@SherazKhan FYI you don't ever really need to open more than one pull request. You can always push-force your changes to the branch that you originally used to open the PR.

File: doc/whats_new.rst [hunk 1]
```diff
@@ -22,6 +22,8 @@ BUG
 
     - Fix computation of AR coefficients across channels in :func:`mne.time_frequency.fit_iir_model_raw` by `Eric Larson`_
 
+    - Fixed maxfilter channel names extra space bug :func:``maxwell.py`` and func:``test_maxwell.py`` are updated by `Sheraz Khan`_
```
Comment (Eric89GXL) [inline doc/whats_new.rst]: single-backticks and complete function name (at least to the submodule level) for linking as:
```
:func:`mne.preprocessing.maxwell_filter`
```
No need to mention changes to `test_maxwell.py`

Comment (Eric89GXL) [inline doc/whats_new.rst]: Also, this is a Sphinx (auto doc build) link:
```
`Sheraz Khan`_
```
You need to give it some URL to link to. See the bottom of the file for examples, and add your name down there.

If you are curious about how this looks, the easiest thing is to look at the CircleCI output. Go to the circle site using the link in the CI list at the bottom of the PR, log in to CircleCI using GitHub (or other method), click "Artifacts", and navigate to the `mne-python/doc/_build/html/dev/whats_new.html` and see:

https://2757-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/whats_new.html

You can see the three errors related to bad links.

**Comment by Eric89GXL** [comment]
@SherazKhan thanks for sticking with this despite the high volume of comments for such a small PR. There are lots of little Git/style/Sphinx things to straighten out that get substantially easier (hopefully) with each subsequent PR

**Comment by SherazKhan** [comment]
Thanks @Eric89GXL on it, hopefully things will be easier with subsequent PR :)

**Comment by agramfort** [comment]
thx @SherazKhan !

congrats for your first PR to mne-python :)

I hope it's the beginning of a long series :)

**Comment by SherazKhan** [comment]
Thanks @agramfort,  at-least I will not secretly stalking mne-python anymore :)

**Pull request merged**